californium-proxy/pom.xml

```diff
@@ -41,10 +43,20 @@
 		</dependency>
 		<dependency>
 			<groupId>org.apache.httpcomponents</groupId>
-			<artifactId>httpclient</artifactId>
-			<version>${httpcomponents.version}</version>
+			<artifactId>httpasyncclient</artifactId>
+			<version>${httpasyncclient.version}</version>
 			<scope>compile</scope>
 		</dependency>
+		<dependency>
+			<groupId>net.sourceforge.streamsupport</groupId>
+			<artifactId>streamsupport</artifactId>
```

Comment:
GPL! 
I'm not sure, if you read the ECA carefully, but I guess using GPL is not possible :-)

Reply:
I've read through it briefly (and now, yet again) and to my understanding they are talking about including source code, not referencing through maven. This is more discussed here: https://www.eclipse.org/legal/eplfaq.php#USEINANOTHER What do you think?

Reply:
Thanks for making me aware of this, I'll try and find a workaround :)